packages/core/scripts/liquidity-mining/yusdeth-weekly-payouts/Week_5_Mining_Rewards.json

```diff
@@ -0,0 +1,539 @@
+    "0xc35ab820c9acefadfd96b4f4ea03a97c70a2d2ac": "77.32652662556136823",
+    "0xc5aa216ab58291b7cfe9be87c0e46917de032907": "2.994909772558055676",
+    "0xc5cce59bb6173ad6fda377ee8a99f07415d57cba": "0.415909646271707044",
+    "0xc8fc213ecb824d0b3083449156864b905d1a818c": "1.301616659651966595",
+    "0xca7c767854b1e7305de07247ab85e30543d1d9c9": "63.789787931309577941",
+    "0xcdee22eed0712f4d00a5b0ac3507cc8aad64ac08": "0.196282934438396284",
+    "0xd0b23f8e6d6f0fb85dafa9c6d2dc4a7f73849b39": "0.788710945973975949",
+    "0xd0dc07b98769f23a7bdbef15a35faa256cb65dcf": "0.896727318820752538",
+    "0xd18001f022154654149ed45888c9c29def6d3ce6": "1.91321648033336432",
+    "0xd38d3b64504480945f482c03670352ca35679630": "1.084280765687517569",
+    "0xd711193c2068575bcfadf3d51b98a0f573cc6492": "128.460229932952497812",
+    "0xd8827d60486e01a6c54f7fc2ddcd9527a26af8ba": "2.642830636151963706",
+    "0xdaf3f7f611efa83033155d23c3ebc5b52ddbaebb": "7.534067123777911181",
+    "0xdc808644dc935dda815742a5adfae3860617261f": "0.826138342087609652",
+    "0xdd10664a1f77c915048908587e6961fbf9d0813e": "1.315394637737308714",
+    "0xdd6fc573cf73701aa31834644e245d9fb5bf906d": "162.640355792573428764",
+    "0xde1b62535e31561c78ba20e49053f2dbcff4b9cd": "0.132250226687644146",
+    "0xde8d36b1a157dbd62ab9983f82dd85b1ecc92025": "4.174268015759747018",
+    "0xe07ef543c16431275718cc1f114b0241ff217278": "0.170396806797418096",
+    "0xe1de283eab72a68f7ff972fca13f8953c6e15e51": "0.225266847865628345",
+    "0xe233cec774061e9871b865c59d1df66be52e3388": "2.758489083103908434",
+    "0xe2d128323cf7560a6e7a82726d7b425aedc7a556": "0.731557712884991048",
+    "0xe38ef7739775ec8c9ad62b759eb6aa9f047d84ed": "0.273672777775708429",
+    "0xe5a3c7a4857b64ef2199889741705db743bc166f": "0.557705291034929319",
+    "0xe765e0c5f5e6acea5175a7da83b88c5dc0f2b437": "1.240851947681654306",
+    "0xe925445ec68d6a9ce15567e1f769fd481ce9bae1": "0.263877436341895825",
+    "0xef22364cdabd7f39a3b6fd836f059336c810a8a9": "0.065842787677608424",
+    "0xf0a8cea0de817eec61b4d27456cfb003167e8024": "1.473425255001572468",
+    "0xf0f32c3c8def7b43caa14fed5752abc5381c8374": "9.594895798756016362",
+    "0xf307077dd0e27a382e93f2e3d13a9c6584582332": "0.166169190239254803",
+    "0xf4290ab1a8129b8b887d3ab331f24ecf812bda8e": "1.302635941705406792",
+    "0xf56d610cbf3208ff6f009cb740beff4e9ef4d7ad": "1.122148503508754744",
+    "0xf59e58ff8b8ba7cbbb22e583b2d4be309e0adbd3": "2.242651556317490551",
+    "0xf627e5f4bad95a956468d8bb6ee20b119f992e96": "3.421881820741333588",
+    "0xf8ad2c3f153c1f9587643ba3698a284d32f9cf6e": "77.641219443771686895",
+    "0xfbb42577eb5e6c40af214bc457afe1922f55556b": "0.21552582381996938",
+    "0xfe62ad2613ed201459348b3e8cf9cc430be02c23": "5.30487417705070542",
+    "0xff3fc772434505abff38eecde3c689d4b0254528": "5.404054226582761962",
+    "0xadeeb9d09b8bcee10943198fb6f6a4229bab3675": "41.720449000633482295"
+  },
+  "weeklyShareHolderPayoutBeforeRollJoin": {
```

Comment:
How does this section differ from the one above that's just called shareHolderPayout (is the first one a superset?)?

Reply:
yup, this is a superset. this `weeklyShareHolderPayoutBeforeRollJoin` contains the output of the non-roll period before joining the roll period. it's kept for auditability.

Reply:
I updated the PR description to explain the different file outputs.